buttons/src/main/java/com/decathlon/vitamin/buttons/VitaminButtons.kt

```diff
@@ -4,38 +4,86 @@ import android.content.Context
 import android.util.AttributeSet
 import com.google.android.material.button.MaterialButton
 
-open class VitaminPrimaryButton @JvmOverloads constructor(
```

Comment:
What about a default Kotlin class for the medium or large implementation? (of course, it should be documented)

Reply:
Note that if you aren't agree about this solution, we should keep default classes for now and add a Deprecated annotation on their to avoid any breaking changes between two minor versions and/or not communicated to our consumers earlier.

Reply:
Fixed